backend/selectgen.ml

```diff
@@ -119,7 +78,6 @@ let env_find_static_exception id env =
 
 let env_enter_trywith env kind =
   match kind with
-  | Regular -> { env with trap_stack = Generic_trap env.trap_stack; }
   | Delayed id -> let env, _ = env_add_static_exception id [] env in env
```

Comment:
(Can we delete the `Generic_trap` constructor?
I think it is no longer produced, only matched upon.)

Reply:
Ah yes, deleting that now